examples/question_answer/README.md

```diff
@@ -0,0 +1,144 @@
+# Astarte Question Answer example
+
+This example will guide you through setting up a simple ESP32 application running the Astarte SDK
+and interacting with a remote instance of Astarte.
+
+We will configure the device to perform the following actions:
+- Connect to a remote Astarte instance using a manually generated Device ID and Credential Secret.
+- When a "question" message is received from the remote instance of Astarte reply with an
+"answer" message from the device.
+
+If you want to start using a local instance of Astarte hosted on your machine and register the
+device automatically you can check out the [`toggle_led` example](../toggle_led) in this repository.
+
+## Prerequisites
+
+Three elements are required to run this example:
+- An installation of the [ESP-IDF](https://docs.espressif.com/projects/esp-idf/en/stable/esp32/).
+- A remote instance of Astarte. We will assume you created your own realm on
+[Astarte Cloud](https://astarte.cloud/) and you will be managing it through the
+[Astarte Dashboard](https://github.com/astarte-platform/astarte-dashboard).
+- The [astartectl](https://github.com/astarte-platform/astartectl/releases) tool. This tool is an
+alternative to Astarte Dashboard to manage your remote Astarte instance. It includes some additional
+functionality used in this example.
+
+**NB.** This SDK does NOT support ESP-IDF version 5. Earlier versions 4 and 3 are supported.
```

Comment:
I can see that you are working on the support of ESP-IDF 5 [here](#99). I suppose that it will be supported in a near future.
Consider to rephrase the sentence to make clear that the lack of support is temporary and will be added in the future.